apps/chat/src/components/Marketplace/ApplicationCard.tsx

```diff
@@ -134,7 +217,21 @@ export const ApplicationCard = ({
         },
       },
     ],
-    [entity, onPublish, t, onDelete, isMyEntity, onEdit],
+    [
```

Comment:
React Hook useMemo has an unnecessary dependency: 'selectedTab'. Either exclude it or remove the dependency array